### DISCUSSION TITLE
Question: Installing on AWS EC2 or Cloud Providers

### DISCUSSION THREAD

**Discussion by jauyzed** [crc-org/crc#3348] [Q&A] [answered]
Hello,

I was curious to know what EC2 instance types are supported for installing CRC? Is it only Bare metal instances? 

Not sure if this has been answered before or maybe I have missed the documentation on what different cloud providers are supported. 

Thanks!

**Comment by gbraad** [comment, marked as answer]
You could try using nested virtualization... but this is NOT recommended. Better to run on the platform directly.

AWS does IIRC not work with EFI GPT partition tables. They still use the older style and therefore converting our image would not work.

It is however possible to run on GCP, but this is not publicly described as this will have other issues when not considering security. All images share certificates and keys, so it is not supposed to be considered as a semi-official way to run a public cluster.

**Comment by lmilbaum** [comment]
Are you aware of any other option to deploy minimal footprint Openshift or Openshift Local on AWS  not using ROSA?
